setup.py

```diff
@@ -11,45 +8,29 @@
     combine_commands,
     get_version,
 )
-
-from setuptools import setup
+import setuptools
 
 
 LONG_DESCRIPTION = 'A Python/ThreeJS bridge utilizing the Jupyter widget infrastructure.'
 
-here = os.path.dirname(os.path.abspath(__file__))
+HERE = Path(__file__).parent.resolve()
 name = 'pythreejs'
-version = get_version(os.path.join(here, name, '_version.py'))
+lab_path = (HERE / name / "labextension")
 
+version = get_version(HERE / name / '_version.py')
 
 cmdclass = create_cmdclass(
     'js',
     data_files_spec=[
-        ('share/jupyter/nbextensions/jupyter-threejs',
-         name + '/static',
-         '*.js'),
-        ('share/jupyter/nbextensions/jupyter-threejs',
-         name + '/static',
-         '*.js.map'),
-        ('share/jupyter/lab/extensions',
-         'js/lab-dist',
-         'jupyter-threejs-*.tgz'),
```

Comment:
i guess we still want lab2 support?

Reply:
Yes, I suppose we do (please excuse the slash-and-burn approach :P)